_build/templates/default/sass/index.scss

```diff
@@ -1805,3 +1805,23 @@ iframe[classname="x-hidden"] {
     margin-top: 0 !important;
   }
 }
+
+::-webkit-scrollbar, ::-webkit-scrollbar-thumb {
+  width: 1rem;
+  height: 1rem;
+  border: .25rem solid transparent;
+  border-radius: .5rem;
+  background-color: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  box-shadow: inset 0 0 0 1rem rgba(85,108,136,.2);
```

Comment:
It would be nice if we could [use an existing](https://github.com/modxcms/revolution/blob/3.x/_build/templates/default/sass/_colors-and-vars.scss) or add a new variable for at least the color.